frontend/src/bundles/landing/components/button-tabs/button-tab.tsx

```diff
@@ -0,0 +1,23 @@
+import classNames from 'classnames';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+    title: string;
+    isActive: boolean;
+    disabled: boolean;
+};
+
+const ButtonTab: React.FC<Properties> = ({ title, isActive, disabled }) => {
+    const tabClassName = classNames(styles.tab, {
+        [styles.active]: isActive,
+        [styles.disabled]: disabled,
+    });
+    return (
+        <button disabled={disabled} className={tabClassName}>
+            {title}
```

Comment:
I think it should be 'label'

Reply:
```suggestion
            {label}
```

Reply:
Resolved in [52f5038](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/189/commits/52f5038f8266bba4cd7f2479a2d33a26612d7116)